src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,64 @@
-export const Tabs = () => {};
+import { FC, useState } from 'react';
+import classNames from 'classnames';
+import { Tab } from './types/interface';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: FC<Props> = ({ tabs, selectedTabId, onTabSelected }) => {
+  let currentTab = tabs.find(({ id }) => id === selectedTabId);
+
+  if (!currentTab) {
+    const [tab] = tabs;
+
+    currentTab = tab;
+  }
+
+  const [text, setText] = useState(currentTab.content);
+  const choseTab = (tab: Tab) => {
+    const { content, id } = tab;
+
+    setText(content);
+
+    if (id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames({
+                  'is-active': id === currentTab?.id,
+                })}
```

Comment:
Replace `currentTab?.id` with `selectedTabId` to determine the active tab based on the provided prop.

Reply:
I can't change currentTab?.id with selectedTabId because this test fails:
<img width="1115" alt="image" src="https://user-images.githubusercontent.com/106397016/234871127-ee40b1b7-784a-4aae-b310-bbc57b76c5c4.png">